apps/catalogue/src/views/cohorts/CohortView.vue

```diff
@@ -0,0 +1,298 @@
+<template>
```

Comment:
This already starts to look good, I am curious what you think? We could generalize some of these blocks to simply pass a list of columns and then they show all of them as key-value but I am not sure that will help a lot (my thoughts: we could do this for some sections in a generic way so when adding columns we don't need to change template)

Reply:
no, thats actually it what i started with ( and what is build is the resource view ) it does not allow for a high quality interface as that has specific details ( combining various fields in a specific context ).Having views and accompanied queries allows for this ( and easy editing by data managers ). 

Doing the rendering with reusable components ( that have no knowledge of a schema), ensures we can have a consistent interface across the emx system. And allows third party contributors of work on these components. The live preview of component editing would need to be improved imo ( as in edit the component style and view it in the context of some app)